internal/utils/file_type.go

```diff
@@ -0,0 +1,156 @@
+package utils
+
+import (
+	"github.com/gabriel-vasile/mimetype"
+	"io"
+	"mime"
+	"path/filepath"
+	"strings"
+)
+
+type FileType int
+
+const (
+	Unknown FileType = iota
+	Application
```

Comment:
Maybe prefix this `FTApplication` since it will be imported as `utils.Application` which may be confusing...